source/tank/controller/move/DynamicMovementController.hx

```diff
@@ -50,20 +41,17 @@ class DynamicMovementController extends BaseController implements IMoveControlle
 		}
 	}
 
-	private function getNewDirection():FlxPoint {
-		var newDirection:FlxPoint = null;
-		var newX = FlxG.random.int(-1, 1);
-		var newY = FlxG.random.int(-1, 1);
-		trace(newX);
-		trace(newY);
-		return FlxPoint.weak(SPEED * newX, SPEED * newY);
+	private function rotateAmount():FlxPoint {
+		var angle = FlxG.random.int(-45, 45, noTinyRotations);
+
+		return controlledTank.velocity.rotateByDegrees(angle);
 	}
```

Comment:
this functionality is super cool, I like how the tank really does feel like it's zeroing in on me even though it's randomly patrolling around.